source/common/json/config_schemas.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+namespace Json {
+
+class Schema {
+public:
+  // Listener Schemas
+  static const std::string LISTENER_SCHEMA;
+
+  // Network Filter Schemas
+  static const std::string CLIENT_SSL_SCHEMA;
```

Comment:
From a naming perspective I mentioned to rename them to include NETWORK_FILTER. The other option is to move this into a sub-struct called NetworkFilter. Either way.